packages/desktop/views/dashboard/settings/views/profile/ProfileSettings.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+    import { activeProfile } from '@core/profile'
+    import { ProfileSettingsRoute } from '@core/router'
+    import features from '@features/features'
+    import { HR } from 'shared/components'
+    import { ChangeProfileName, Currency, DeleteProfile } from './'
+
+    const settings: {
+        component: unknown
+        childRoute: ProfileSettingsRoute
+        requireLogin?: boolean
+    }[] = [
```

Comment:
Should use an interface

Reply:
Copy and pasted will be refactored when we merge settings to generic component.